### DISCUSSION TITLE
Strange (void)io; in example_glfw_opengl3

### DISCUSSION THREAD

**Discussion by AntMaster7** [ocornut/imgui#4733] [New Users Build/Link/Run issues ONLY!]
I noticed the somewhat strange statement `(void)io;` in line 67 [here](https://github.com/ocornut/imgui/blob/master/examples/example_glfw_opengl3/main.cpp). Casting something to void might have its reasons. But in the given case, the value is not assigned anywhere. Does this statement have a reason or is just some leftover?

**Comment by ocornut** [comment]
This is in order to avoid a “unused variable” warning while conveniently allowing to uncomment the config lines below.

We should use a IM_UNUSED() macro to make this more obvious.
